test/models/bot/alegre_test.rb

```diff
@@ -35,6 +41,7 @@ def setup
       WebMock.stub_request(:get, 'http://alegre/text/langid/').to_return(body: {
         'foo': 'bar'
       }.to_json)
+      Bot::Alegre.stubs(:request_api).raises(RuntimeError)
```

Comment:
Please make sure to `unstub` at the end of the test (or at once in `def teardown`) because it doesn't happen automatically in this version of the library. Something like: `Bot::Alegre.unstub(:request_api)`.